elements/lisk-cryptography/src/hash.ts

```diff
@@ -43,3 +43,13 @@ export const hash = (data: Buffer | string, format?: string): Buffer => {
 		'Unsupported data format. Currently only Buffers or `hex` and `utf8` strings are supported.',
 	);
 };
+
+export const verifyChecksum = (
```

Comment:
maybe if it's this simple, then probably we don't need to add this function.
This is not really specific to checksum either

Reply:
ok then I will not add the changes in cryptography 👍 
it will not be useful in general so